From b111c80f35027ade1af169a024b680d18bc4ceb7 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Wed, 6 May 2015 19:23:01 +0800
Subject: [PATCH 1062/1594] MLK-10854 dts: imx6ul: add missed dtbs into
 dts/Makefile

commit 24d2364452dba42b897342c9a5f73a3dc3c3f79d from
git://git.freescale.com/imx/linux-2.6-imx.git

Those dtbs are missed in dts/Makefile.
Then 'make ARCH=arm dtbs' command will miss to compile them.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/boot/dts/Makefile |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 9163240..46698c1 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -363,6 +363,10 @@ dtb-$(CONFIG_SOC_VF610) += \
 dtb-$(CONFIG_ARCH_MXC) += \
 	imx6ul-ddr3-arm2.dtb \
 	imx6ul-ddr3-arm2-flexcan2.dtb \
+	imx6ul-ddr3-arm2-lcdif.dtb \
+	imx6ul-ddr3-arm2-mqs.dtb \
+	imx6ul-ddr3-arm2-spdif.dtb \
+	imx6ul-ddr3-arm2-wm8958.dtb \
 	imx6ul-evk.dtb \
 	imx7d-12x12-lpddr3-arm2.dtb \
 	imx7d-12x12-lpddr3-arm2-enet.dtb \
-- 
1.7.5.4

